package.json

```diff
@@ -13,15 +13,15 @@
   },
   "author": "Canonical webteam",
   "devDependencies": {
-    "autoprefixer": "9.4.7",
-    "cookie-policy": "1.0.6",
+    "autoprefixer": "9.4.10",
+    "cookie-policy": "1.1.0",
     "node-sass": "4.11.0",
-    "postcss-cli": "6.1.1",
+    "postcss-cli": "6.1.2",
     "sass-lint": "1.12.1",
     "watch-cli": "0.2.3"
   },
   "dependencies": {
     "vanilla-framework": "1.8.1",
-    "global-nav": "2.0.0"
+    "global-nav": "2.0.2"
```

Comment:
it seems I can't import the global nav here: 
```
ReferenceError: canonicalGlobalNav is not defined[Learn More] 8337:297:1

    <anonymous> http://0.0.0.0:8030/the-snapcraft-format/8337:297 
```

Reply:
use: 2.0.3?